### DISCUSSION TITLE
How to replace contents of a list

### DISCUSSION THREAD

**Discussion by Justinzobel** [mikefarah/yq#1926] [Q&A] [answered]
file contents example:
```
animals:
    - bird
    - frog
    - leopard
```

I want to be able to replace the list of animals.

I tried with `yq -i '.animals = "rabbit,donkey,horse"' my.yml` but it just adds it as a value of animals.

**Comment by mikefarah** [comment, marked as answer]
Yes that right, it will replace the contents of `.animals` (which was an array) with the string you have provided.

To get what you want, you need to:

```
yq -i '.animals = ["rabbit", "donkey", "horse"]' my.yml
```

**Reply by Justinzobel** [reply]
Thanks!
